polaris-react/src/components/Navigation/Navigation.module.scss

```diff
@@ -57,14 +57,11 @@ $nav-max-width: 360px;
   flex-direction: column;
   align-items: stretch;
   max-width: 100%;
+  padding-top: var(--p-space-400);
 
   &:focus {
     outline: none;
   }
-
-  @media #{$p-breakpoints-md-up} {
-    padding-top: var(--p-space-400);
-  }
```

Comment:
Could we maybe keep this media query and then add `padding-top: var(--p-space-100)` for other break points? the `400` looks a little too much in the admin 

|space|screenshot|
|-|-|
|`--p-space-400`|<img width="392" alt="Screenshot 2024-01-05 at 1 33 42 PM" src="https://github.com/Shopify/polaris/assets/20652326/01cb405b-869d-4517-a6d0-be5751b5ddcf">|
|`--p-space-100`|<img width="418" alt="Screenshot 2024-01-05 at 1 33 31 PM" src="https://github.com/Shopify/polaris/assets/20652326/2c4a19ac-1ce4-4d1f-b978-f1232deda65f">|